java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -144,6 +150,63 @@ public T set(String key, String value) {
         return getThis();
     }
 
+    /**
+     * Retrieve the value associated with <code>field</code> in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hget/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param field The field in the hash stored at <code>key</code> to retrieve from the database.
+     * @return The value associated with <code>field</code>, or <code>null</code> when <code>field
+     *     </code> is not present in the hash or <code>key</code> does not exist.
+     */
+    public T hget(@NonNull String key, @NonNull String field) {
+        ArgsArray commandArgs = buildArgs(key, field);
+
+        protobufTransaction.addCommands(buildCommand(HashGet, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sets the specified fields to their respective values in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hset/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @param fieldValueMap A field-value map consisting of fields and their corresponding values to
+     *     be set in the hash stored at the specified key.
+     * @return The number of fields that were added.
+     */
+    public T hset(@NonNull String key, @NonNull Map<String, String> fieldValueMap) {
+        ArgsArray commandArgs =
+                buildArgs(
+                        Stream.concat(
+                                        Stream.of(key),
+                                        fieldValueMap.entrySet().stream()
+                                                .flatMap(entry -> Stream.of(entry.getKey(), entry.getValue())))
+                                .toArray(String[]::new));
```

Comment:
Consider creating a util class to combine this and https://github.com/aws/glide-for-redis/pull/955/files#r1488732655 in one place